library/extractor/src/main/java/com/google/android/exoplayer2/extractor/ts/DefaultTsPayloadReaderFactory.java

```diff
@@ -176,11 +182,41 @@ public TsPayloadReader createPayloadReader(int streamType, EsInfo esInfo) {
       case TsExtractor.TS_STREAM_TYPE_DVBSUBS:
         return new PesReader(
             new DvbSubtitleReader(esInfo.dvbSubtitleInfos));
+      case TsExtractor.TS_STREAM_TYPE_AIT:
+        return new SectionReader(new SectionPassthrough(MimeTypes.APPLICATION_AIT));
       default:
         return null;
     }
   }
 
+  public class SectionPassthrough implements SectionPayloadReader {
```

Comment:
Can you add some documentation about what this class is for/what it does?

Reply:
Added